sdk/src/main/java/com/hedera/hashgraph/sdk/FeeComponents.java

```diff
@@ -0,0 +1,155 @@
+package com.hedera.hashgraph.sdk;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+
+
+
+public class FeeComponents {
+    
+    
+    private long min; // A minimum, the calculated fee must be greater than this value
+    private long max; // A maximum, the calculated fee must be less than this value
+    private long constant; // A constant contribution to the fee
+    private long bpt; // The price of bandwidth consumed by a transaction, measured in bytes
+    private long vpt; // The price per signature verification for a transaction
+    private long rbh; // The price of RAM consumed by a transaction, measured in byte-hours
+    private long sbh; // The price of storage consumed by a transaction, measured in byte-hours
+    private long gas; // The price of computation for a smart contract transaction, measured in gas
+    private long tv; // The price per hbar transferred for a transfer
+    private long bpr; // The price of bandwidth for data retrieved from memory for a response, measured in bytes
+    private long sbpr; // The price of bandwidth for data retrieved from disk for a response, measured in bytes
+
+
+
+    public FeeComponents() {
+        // All members initialized to 0
+    }
+
+
+    // setters
+
+    public FeeComponents setMin(long min) {
```

Comment:
Use one new line after each method, but also can you rearrange these getters and setters to be in groups of get/set for each field.
```java
    public long getMax() {
        return max;
    }

    public FeeComponents setMax(long max) {
        this.max = max;
        return this;
    }

    public long getBpt() {
        return bpt;
    }

    public FeeComponents setBpt(long bpt) {
        this.bpt = bpt;
        return this;
    }
```